draft/0004-user-sign-in-experience.md

```diff
@@ -0,0 +1,453 @@
+---
+Start date: 2024-05-24
+Author(s): [simeng-li]
+Revision: 1
+---
+
+# Sign-in experience and experience APIs
+
+## 1. Abstract
+
+Sign-in experience (SIE) is the process of user authentication in the Logto platform. It includes the user interface and experience APIs that allow end-users to interact with the Logto platform for identity verification and profile management.
```

Comment:
btw, i see different words like "Logto platform" and "Logto system" which may be redundant. i use "Logto" in the rest of the comments for simplicity and consistency.